File: 2609/Main.java
import java.util.Scanner;
import java.util.ArrayList;

public class Main {
	public static void main(String [] args ) {
		Scanner sc = new Scanner(System.in);
		int first = sc.nextInt();
		int second = sc.nextInt();
		int a = Divider(first, second);
	}

	public static int Divider(int first, int second) {
		ArrayList<Integer> arr1 = new ArrayList<Integer>();
		ArrayList<Integer> arr2 = new ArrayList<Integer>();

		for (int i = 1 ; i <= first ; i++ ) {
			if (first % i == 0 ) {
				arr1.add(i);
			}
		}

		for (int j = 1; j <= second ; j++) {
			if (second % j == 0 ) {
				arr2.add(j);
			}
		}
		
		int last = arr1.size();
		int best = 0;
		for (int i = last -1 ; i >= 0 ; i-- ) {
			best = arr1.get(i);
			

		return 1;
	}
}

File: 2609/main.cpp
#include <iostream>

using namespace std;

int main(void) {
	int first , second;

	cin >> first >> second;

	int arr1[10001];
	int arr2[10001];

	for (int i = 1 ; i <= first ; i++ ) {
		if ( first % i == 0 ) {
			arr1[i] = 1;
		}
	}

	for (int i = 1 ; i <= second ; i++ ) {
		if ( second % i == 0 ) {
			arr2[i] = 1;
		}
	}

	int maxDiv = 0;

	for (int i = 10000 ; i >= 0 ; i--) {
		if ( arr1[i] != 0 && arr2[i] != 0 ) {
			maxDiv = i;
			break;
		}
	}

	int minMul = first * second;
	minMul /= maxDiv;
	cout << maxDiv << endl << minMul ;

}
